scripts/qa_crowbarsetup.sh

```diff
@@ -2517,7 +2517,13 @@ function onadmin_addupdaterepo()
     if [[ -n "$UPDATEREPOS" ]]; then
         local repo
         for repo in ${UPDATEREPOS//+/ } ; do
-            wget --progress=dot:mega -r --directory-prefix $UPR --no-parent --no-clobber --accept x86_64.rpm,noarch.rpm $repo || exit 8
+            wget --progress=dot:mega \
+                -r --directory-prefix $UPR \
+                --no-parent \
+                --no-clobber \
+                --accept x86_64.rpm,noarch.rpm \
+                ${repo%/}/ \
```

Comment:
@jdsn Good catch for noticing that, although a) it's not strictly necessary, since I think `wget` / Apache should cope with a double-slash fine, and b) it relies on the `extglob` option being set.  Can we safely assume that?  If we have to set `extglob` here, how do we know it won't have an adverse effect on other parts of the code?  Maybe the only safe way is to check the status of the option, and then restore it to the previous state after using it.  #shellsucks